inf-clojure.el

```diff
@@ -940,6 +940,25 @@ prefix argument PROMPT-FOR-SYMBOL, it prompts for a symbol name."
                (inf-clojure-symbol-at-point))))
     (comint-proc-query (inf-clojure-proc) (format (inf-clojure-var-source-form) var))))
 
+(defvar inf-clojure--dump-file-name ".inf-clojure.dump"
```

Comment:
I was referring to the `nrepl-toggle-message-logging`. I don't like putting such logs in a file (by default), as refer to old logs extremely rarely. They are most useful for debug purposes right away. It's trivial to make a buffer scroll automatically when content is added to it.